src/System.Data.SqlClient/src/System/Data/ProviderBase/DbConnectionPoolGroup.cs

```diff
@@ -223,13 +223,6 @@ internal DbConnectionPool GetConnectionPool(DbConnectionFactory connectionFactor
                                 Debug.Assert(PoolGroupStateActive == _state, "state should be active since a pool exists and lock holds");
                             }
                         }
-
-                        if (null != newPool)
```

Comment:
After the change, only one thread can create and add connection pool to pool group. No need to shutdown extra pools since they do not exist anymore.

Reply:
Is this shutdown call still needed if we don't call Startup()? Like at https://github.com/geleems/corefx/blob/c3d680626c49cac367556dc88d13ac23df2190db/src/System.Data.SqlClient/src/System/Data/ProviderBase/DbConnectionPoolGroup.cs#L217 for example